source/sync/data-access-patterns/flexible-sync.txt

```diff
@@ -212,3 +213,50 @@ sync objects with the backend {+app+} and can watch for and react to changes.
 
       To create queries from the .NET Client SDK, see the :ref:`.NET SDK 
       guide to Flexible Sync <dotnet-flexible-sync>`.
+
+Optimize Sync with Asymmetric Sync
```

Comment:
```suggestion
Optimize Sync with Asymmetric Sync (Optional)
```

Just to be clear to Flexible Sync users that they don't have to use Asymmetric Sync